src/evict/evict_page.c

```diff
@@ -95,12 +95,14 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
     WT_DECL_RET;
     WT_PAGE *page;
     uint64_t time_start, time_stop;
-    bool clean_page, closing, inmem_split, local_gen, tree_dead;
+    uint32_t session_flags;
+    bool clean_page, closing, inmem_split, local_gen, tree_dead, is_owner;
 
     conn = S2C(session);
     page = ref->page;
     closing = LF_ISSET(WT_EVICT_CALL_CLOSING);
     local_gen = false;
+    session_flags = 0;
```

Comment:
Since this is only used in one place, I'd move the initialization to that place. (And, since it's returned by `__wt_hs_cursor()`, it actually shouldn't need initialization??)